internal/exportimport/database/exportimport.go

```diff
@@ -369,6 +369,19 @@ func (db *ExportImportDB) CreateFiles(ctx context.Context, ei *model.ExportImpor
 			logger.Debugw("scheduled new export file for importing", "exportImportID", ei.ID, "filename", fname)
 			insertedFiles++
 		}
+
+		// Close any files that aren't in the list that are retrying.
+		_, err = tx.Exec(ctx, `
```

Comment:
it would be good to return the number of files that moved into a failed state and log this information